stellarphot/photometry/tests/test_photometry.py

```diff
@@ -514,14 +535,34 @@ def test_photometry_on_directory():
 
         expected_deviation = np.pi * aperture**2 * noise_dev
 
-        # We could require that the result be within some reasonable
-        # number of those expected variations or we could count up the
-        # actual number of background counts at each of the source
-        # positions.
+        # We have two cases to consider: use_coordinates="sky" and
+        # use_coordinates="pixel".
+        if coords == "sky":
+            # In this case, the expected result is the test below.
 
-        # Here we just check whether any difference is consistent with
-        # less than the expected one sigma deviation.
-        assert np.abs(expected_flux - obs_avg_net_cnts) < expected_deviation
+            # We could require that the result be within some reasonable
+            # number of those expected variations or we could count up the
+            # actual number of background counts at each of the source
+            # positions.
+
+            # Here we just check whether any difference is consistent with
+            # less than the expected one sigma deviation.
+            assert np.abs(expected_flux - obs_avg_net_cnts) < expected_deviation
+        else:
+            # In this case we are trying to do photometry in pixel coordinates,
+            # using the pixel location of the sources as found in the first image --
+            # see the line where found_sources is defined.
+            #
+            # However, the images are shifted with respect to each other by
+            # list_of_fakes, so there are no long stars at those positions in the
```

Comment:
"no long" --> "no longer"?

Reply:
🤣 I wish we could take care of collimation issues in the software! Will fix in the next batch of changes.